packages/mobile/src/account/LinkBankAccountScreen.tsx

```diff
@@ -251,8 +284,10 @@ export function StepOne() {
 export function StepTwo() {
   const { t } = useTranslation()
   const finclusiveKycStatus = useSelector(finclusiveKycStatusSelector)
+  const finclusiveRegionSupported = useSelector(finclusiveRegionSupportedSelector)
   const plaidParams = useSelector(plaidParamsSelector)
-  const stepTwoEnabled = useSelector(linkBankAccountStepTwoEnabledSelector)
+  const stepTwoEnabled =
```

Comment:
interesting.. so this means when the user's geo isn't supported they'll see "We successfully verified your identity. We'll let you know when you can finish linking your bank account."

Reply:
one option could be to refactor the selector `linkBankAccountStepTwoEnabledSelector` to return the value of the feature flag && finclusiveRegionSupported

Reply:
since I'm not sure there's a use case anymore for just checking the value of the feature flag in isolation